docs/src/content/docs/guides/release.mdx

```diff
@@ -3,8 +3,6 @@ title: Release process
 description: Learn how we version and release the SDK and recent changes.
 ---
 
-import { Content as AgentsSdkChangelog } from '../../../../../packages/agents/CHANGELOG.md';
```

Comment:
This is unused and can cause an error on the translated page side. I could fix the error, but just removing it is easier.